common/constant.js

```diff
@@ -1,30 +1,27 @@
-"use strict";
-
-var amazon = require('common/makeCredential')('amazon');
-var CONSTANT = {
+const CONSTANT = {
```

Comment:
__[jshint]__
*  'const' is available in ES6 (use esnext option) or Mozilla JS extensions (use moz).